### PR TITLE
ADEN-6007 test veles events

### PR DESCRIPTION
**Pull request by bulaszenko**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsPrebidObject.java [hunk 1]
```diff
@@ -67,6 +75,27 @@ public void goToNextPageView() {
     waitTitleChangesTo(NEXT_TITLE);
   }
 
+
+  public void verifyVelesIncontentSlotDisplayedOasis(){
```
Comment (fraszczakszymon) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsPrebidObject.java]: This is not related with prebid. Can we move it to some other/new class?

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPrebidVelesOasis.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsPrebidObject;
+import org.testng.annotations.Test;
+
+public class TestAdsPrebidVelesOasis extends TemplateNoFirstLoad {
```
Comment (fraszczakszymon) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPrebidVelesOasis.java]: Remove Prebid please

**Comment by fraszczakszymon** [comment]
Remove all prebid occurrences please. Veles is not using prebid anymore :)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+
+public class AdsVelesObject extends AdsBaseObject {
+
+  private static final String INCONTENT_VIDEO = ".video-display-wrapper";
+
+  public static final String SECOND_ARTICLE_HEADER_OASIS = "#Player_above";
+  public static final String FIRST_ARTICLE_HEADER_MOBILE = "#Header";
+
+  public final String BIDDER_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_with_fallback_bid.*";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java]: MAJOR Make this final field static too. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
MAJOR Rename this field "BIDDER_PLAYER_EVENT_PATTERN" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  public final String DIRECT_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_with_direct.*";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java]: MAJOR Make this final field static too. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
MAJOR Rename this field "DIRECT_PLAYER_EVENT_PATTERN" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  public final String NO_OFFER_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_without_offer.*";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java]: MAJOR Make this final field static too. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
MAJOR Rename this field "NO_OFFER_PLAYER_EVENT_PATTERN" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestVideoVelesMobile.java [hunk 6]
```diff
@@ -0,0 +1,50 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsPrebidObject;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestVideoVelesMobile.java]: MINOR Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsPrebidObject'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVideoVelesOasis.java [hunk 7]
```diff
@@ -0,0 +1,47 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsPrebidObject;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVideoVelesOasis.java]: MINOR Remove this unused import 'com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsPrebidObject'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 8 issues
* ![MAJOR][MAJOR] 6 major
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+public class AdsVelesObject extends AdsBaseObject {
+
+  private static final String INCONTENT_VIDEO = ".video-display-wrapper";
+
+  public static final String SECOND_ARTICLE_HEADER_OASIS = "#Player_above";
+  public static final String FIRST_ARTICLE_HEADER_MOBILE = "#Header";
+
+  public static final String BIDDER_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_with_fallback_bid.*";
+  public static String DIRECT_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_with_direct.*";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java]: MAJOR Make DIRECT_PLAYER_EVENT_PATTERN a static final constant or non-public and provide accessors if needed. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
MAJOR Make this "public static DIRECT_PLAYER_EVENT_PATTERN" field final [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1444)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+  public static String NO_OFFER_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_without_offer.*";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java]: MAJOR Make NO_OFFER_PLAYER_EVENT_PATTERN a static final constant or non-public and provide accessors if needed. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
MAJOR Make this "public static NO_OFFER_PLAYER_EVENT_PATTERN" field final [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1444)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues
* ![MAJOR][MAJOR] 4 major
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 4 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+  public static final String BIDDER_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_with_fallback_bid.*";
+  public static final String DIRECT_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_with_direct.*";
+  public static final String NO_OFFER_PLAYER_EVENT_PATTERN = ".*adengplayerinfo.*event_name=in_viewport_without_offer.*";
+
+  public AdsVelesObject(WebDriver driver, String testedPage) {
+    super(driver, testedPage);
+  }
+
+  public void verifyVelesIncontentSlotDisplayedOasis(){
+    tiggerIncontentPlayerOnOasis();
```
Comment (fraszczakszymon) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsVelesObject.java]: Do we need separate method for that? Can't we scroll to some INCONTENT_VIDEO selector?
